src/PageContent/Directory/DirectoryCard.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import "typeface-roboto";
```

Comment:
Change to Open Sans. In my PR #6 I added `<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">` into index.html so you shouldn't have a problem rendering that font once my pr is merged.